Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/Security - Add-AuthenticatedUsersToRemoteDesktopUsers.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Adds the Authenticated Users local group to the local Remote Desktop Users group so that any authenticated user may connected via Remote Desktop. By default, only Administrators can connect.",
+  "description": "What is it: Adds the Authenticated Users local group to the local Remote Desktop Users group so that any authenticated user may connected via Remote Desktop. \n Simeon Notes: Allows users to access their computers via Remote Desktop.",
```

Comment:
**What does this do?**
Adds the Authenticated Users group to the Remote Desktop Users group so any authenticated user can connect via Remote Desktop.

**Why should you use this?**
If you want to allow users to connect remotely via Remote Desktop.

**What is the end-user impact?**
Users may connect remotely via Remote Desktop.

**Learn more**
N/A